maquetado/scss/navbar.scss

```
@@ -0,0 +1,26 @@
+.header {
+  height: $nav-height;
+}
+.navbar {
+  background-color: white;
+  box-shadow: 0 0 16px $black;
```

```suggestion
  box-shadow: 0 0 16px rgba($black, 0.3);
```
Because is too dark with black only